tools/protobuf-compile/protobuf-compile.go

```diff
@@ -157,7 +159,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	protocGenGoGrpcExec, err := filepath.Abs(protocGenGoExec)
+	protocGenGoGrpcExec, err = filepath.Abs(protocGenGoGrpcExec)
```

Comment:
Just to add a few more details to these changes:

Prior to these changes both `--plugin` flags were passing in protoc-gen-go and protoc-gen-go-grpc was unused.

Now, protoc-gen-go-grpc is being supplied to protoc as a plugin.

However, we're using an old version of protoc-gen-go that includes the ability to generate grpc; I don't think protoc-gen-go-grpc is used regardless.